corehq/apps/app_manager/models.py

```diff
@@ -2713,12 +2713,27 @@ def validate_for_build(self):
         return errors
 
 
+class ReportGraphConfig(DocumentSchema):
```

Comment:
Hrm. I'm wondering if making GraphConfiguration subclass ReportGraphConfig (and rename them, something like GraphConfigFixedData and GraphConfigUserDefinedData ...those are terrible class names, but you get the idea) would make future graphing functionality easier to implement (rather than potentially adding new code in both places). This isn't a blocking comment, just thinking about it.